jenkins-build-pack-website/Jenkinsfile

```diff
@@ -832,6 +836,9 @@ def loadConfigModule(buildModuleUrl){
 	}
 }
 
+/**
+ * Non-lazy JSON parser
+ */
 @NonCPS
 def parseJson(jsonString) {
     def lazyMap = new groovy.json.JsonSlurper().parseText(jsonString)
```

Comment:
JsonSlurper is a lazy parsor.  Use jsonSlurperClassic to avoid lazy map exceptions

Reply:
In all the files JsonSlurper is used. I have reused the same function.

Reply:
@suryajak  @supritat 
I am approving this PR based on Suprita's comment. jsonSlurper is a lazy parser and which can create problems.  I would recommend to use  jsonSlurperClassic across board to avoid such problems.

Reply:
In internal do we no longer use jsonSlurper?